cucumber-rails.gemspec

```diff
@@ -26,14 +26,14 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency('capybara', '>= 3.11', '< 4')
   s.add_runtime_dependency('cucumber', '>= 5', '< 10')
-  s.add_runtime_dependency('railties', '>= 5.2', '< 8')
```

Comment:
Can we make this `< 9`

Reply:
Hi sorry, I've missed this

However, Rails uses a "shifted semver version", so 8.1 should be considered as a major release 

https://guides.rubyonrails.org/maintenance_policy.html#versioning

I will change to `8.1` and a note in the commit message, but if you prefer, I can switch to 9.0

Reply:
Thanks for the info. Let me take a read - Maybe hold off on this change for now then. Thanks for the info

Reply:
Let me read this too. Sorry this is quite a lot of good info and I want to check it all out. Let me get back to you Friday/Monday (That way you're not constantly checking for updates).

Reply:
Hi there. So after consideration and given the lack of response from rails. Let's set this as `< 9` and if things break we will fix them. This gem doesn't maybe get as much active updates so making the maintenance easier is better imo